src/pages/home/sidebar/optionPropTypes.js

```diff
@@ -19,11 +19,14 @@ const optionPropTypes = PropTypes.shape({
     alternateText: PropTypes.string.isRequired,
 
     // List of participants of the report
-    participantsList: PropTypes.arrayOf(participantPropTypes).isRequired,
+    participantsList: PropTypes.arrayOf(participantPropTypes),
```

Comment:
This too. Is there a good reason for this change? Could this cause problems in other places of the app?